<launch>
    <node name="stoplight_watcher" pkg="rr_iarrc" type="stoplight_watcher" output="screen">
      <param name="stoplight_topic" value="/start_detected" />
      <param name="img_topic" value="/camera_center/image_color_rect"/>
    </node>
</launch>
